themeServer.js

```diff
@@ -0,0 +1,76 @@
+const fs = require('fs')
+const os = require('os')
+const express = require('express')
+const bodyParser = require('body-parser')
+
+const ifaces = os.networkInterfaces()
+const PORT = 8090
+const THEME_PATH = './src/theme/variables'
+const envFile = './env.json'
+let address = ''
+let envJSON = { THEME_SERVER: {} }
+
+process.env.THEME_PATH = THEME_PATH
+
+try {
+  envJSON = JSON.parse(fs.readFileSync(envFile, 'utf8'))
+} catch (e) {
+  console.log(e)
+}
+
+try {
+  // Get Local Host Address
+  Object.keys(ifaces).forEach(function (ifname) {
+    let found = false
+
+    ifaces[ifname].forEach(function (iface) {
+      if (iface.family !== 'IPv4' || iface.internal !== false) {
+        // skip over internal (i.e. 127.0.0.1) and non-ipv4 addresses
+        return
+      }
+      if (found) return
+      address = iface.address
+      found = true
+    })
```

Comment:
Instead of a `forEach`, I would use a regular `for` loop. Right now it will still iterate through even though `found` is true. But inside a `for` loop, you could use `break` to stop iterating.

Reply:
Addressed comment.